frontend/src/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Bedrock Claude Chat",
+  "short_name": "Cloude Chat",
+  "start_url": "/index.html",
+  "display": "standalone",
+  "icons": [
+    {
+      "src": "/assets/bedrock-icon.png",
```

Comment:
Since the `assets` folder is treated as part of the codebase, it cannot be used as icons. Please copy the `assets/ML-icon.svg` file to the `public` folder, and update the src to `/ML-icon.svg`.

Reply:
Change performed, however I do not have write access to merge both pull requests.